src/Request.php

```diff
@@ -29,6 +29,14 @@ class Request
         'Razorpay-API'  =>  1    
     );
 
+    /**
+     * Headers to be sent with every http request to the API
+     * @var array
+     */
+    protected static $options = array(
```

Comment:
Check this if static is required or not. Also test if values are getting overriden.